### DISCUSSION TITLE
[1 febbraio 2022 - B] Esercizio 4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#214] [Esami]
**Esercizio 4 (3 punti)**

Usando gli assiomi dell’algebra di Boole, verificare la seguente identità:

![image](https://user-images.githubusercontent.com/69389461/211826274-38abda3a-101c-46cf-af3f-3ff0219b8d61.png)

**Comment by Elia-Belli** [comment]
- $(x \bar z + (\bar y + \bar z)(\bar y + \bar x z) \oplus xy$
$(x \bar z + \bar y) \oplus xy$
$(x \bar z + \bar y)(\bar x + \bar y) + (\overline {x \bar z + \bar y})xy$
$\bar y + ((\bar x + z)y)xy$
$\bar y + xyz$
$(\bar y + x)(\bar y + y)(\bar y + z)$
$\bar y + xz$

**Comment by fraxb04** [comment]
@Elia-Belli 
corretto
